backend.py

```diff
@@ -25,6 +25,17 @@ def search_by_date(filename, date, after=True):
 def search_for_modfication_on_date(filename, date):
 	pass
 
+#^^ Change to search by date?
+
+# Fully deletes a file
+def remove_file(file):

```

Comment:
tested on my machine and both of these functions work :)) you might consider writing tests in test.py if that's useful for you in the future. that way i can see and run your tests. idk up you though.